tutorials/01_requests_click.md

```diff
@@ -13,64 +13,243 @@ Výukové materiály o clicku:
 [naucse.python.cz](http://naucse.python.cz/2017/mipyt-zima/intro/click/),
 [GitHub](https://github.com/pyvec/naucse.python.cz/tree/master/lessons/intro/click).
 
-Úkol
-----
+Úkol - Labelord I
```

Comment:
Můžeme to nechat jen jako Úkol, ať nemusím měnit odkazy s kotvou? Je to prkotina, ale zbytečná pakárna. Díky

Reply:
Jasně, hotovo...